WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderFragment.kt

```diff
@@ -168,4 +193,26 @@ class ReaderFragment : Fragment(R.layout.reader_fragment_layout) {
             }
         }
     }
+
+    private fun showReaderInterests() {
+        val readerInterestsFragment = childFragmentManager.findFragmentByTag(ReaderInterestsFragment.TAG)
+        if (readerInterestsFragment == null) {
+            childFragmentManager.beginTransaction()
+                .replace(
```

Comment:
Just out of curiosity (I have never been sure which methods should be used): Any reason you used "replace" instead of "add" and "commitNow" instead of "commit"?

Reply:
`commitNow` [disallows adding to back stack](https://github.com/aosp-mirror/platform_frameworks_support/blob/2167a9174649d66111f2be5a0b1fc96b15cc48aa/fragment/java/android/support/v4/app/BackStackRecord.java#L671), which is what we want in this case. 

Also it

> [Commits this transaction synchronously.](https://developer.android.com/reference/android/app/FragmentTransaction.html#commitNow()) Any added fragments will be initialized and brought completely to the lifecycle state of their host and any removed fragments will be torn down accordingly before this call returns.

This article [the many flavors of commit ](https://medium.com/@bherbst/the-many-flavors-of-commit-186608a015b1) explains it well
> If you need synchronicity and you are not adding your transaction to the back stack, use commitNow(). 

I have no preference of using `replace` instead of `add`. Found it used in several parts of the existing code and so went ahead with it.